src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -0,0 +1,1204 @@
+/*
```

Comment:
We have the "open question" about keep the cache management classes in cache/ module or in mongoBackend/ module.

Maybe it is better to have them in mongoBackend, as they are very "tied" to the DB technologies (i.e. they use connectionOperationsfunctions and, soon, they will use safeBson functions).

Just a thought...